<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>836</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="buttonHome">
         <property name="text">
          <string>Home</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonTest">
         <property name="text">
          <string>Sink Test</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="2">
            <widget class="QPushButton" name="buttonAddSensor">
             <property name="text">
              <string>+</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0" colspan="4">
            <widget class="QTabWidget" name="tabCameras">
             <property name="currentIndex">
              <number>-1</number>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QPushButton" name="buttonRemoveSensor">
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QCheckBox" name="checkboxDetection">
             <property name="text">
              <string>Detection</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QGroupBox" name="groupBox_4">
             <property name="title">
              <string>Camera-Specific Controls</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="CameraControls" name="cameraControls" native="true"/>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>Overall Device Controls</string>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="1" column="0" colspan="2">
               <widget class="QPushButton" name="buttonRecord">
                <property name="text">
                 <string>Record</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Elapased Time:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="labelElapsed">
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0" colspan="2">
               <widget class="QPushButton" name="buttonPlayback">
                <property name="text">
                 <string>Playback</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QVBoxLayout" name="verticalLayout_4"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>971</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
    <addaction name="actionRestart"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionCameraProperties"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionQuick_Start_Guide"/>
   </widget>
   <widget class="QMenu" name="menuVideo">
    <property name="title">
     <string>Video</string>
    </property>
    <addaction name="actionRecord"/>
    <addaction name="actionOpen_Output_Directory"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionGitHub"/>
    <addaction name="actionCredits"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuVideo"/>
   <addaction name="menuOptions"/>
   <addaction name="menuAbout"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="text">
    <string>Restart</string>
   </property>
  </action>
  <action name="actionRecord">
   <property name="text">
    <string>Record</string>
   </property>
  </action>
  <action name="actionOpen_Output_Directory">
   <property name="text">
    <string>Open Output Directory</string>
   </property>
  </action>
  <action name="actionGitHub">
   <property name="text">
    <string>GitHub</string>
   </property>
  </action>
  <action name="actionCredits">
   <property name="text">
    <string>Credits</string>
   </property>
  </action>
  <action name="actionQuick_Start_Guide">
   <property name="text">
    <string>Quick Start Guide</string>
   </property>
  </action>
  <action name="actionCameraProperties">
   <property name="text">
    <string>Camera Properties</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CameraControls</class>
   <extends>QWidget</extends>
   <header>Widgets/CameraControls/cameracontrols.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
